DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-1.0.0-rc4-004840
\ No newline at end of file
+2.0.0-alpha-5212
```

Comment:
@livarcocc @piotrpMSFT  - looks like we got a bad change in the CLI.  We are no longer padding our commit counts.  We should fix that.